### DISCUSSION TITLE
[Proposal] StateContainer: Please don't wrap child like stacklayout give us the option to make the child extend the child vertical or horizontal like grid

### DISCUSSION THREAD

**Discussion by Strypper** [CommunityToolkit/Maui#945] [New Feature Discussions]
I have been using state-container to switch ContentView adapting tons of UI layout, one of the issues I find with the state container is the children inside it can't stretch to all the available width and height or maybe the state-container design was like stacklayout only take available space when contents are added

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889
